qiita_db/analysis.py

```diff
@@ -869,6 +876,8 @@ def build_files(self, merge_duplicated_sample_ids):
                         dup_samples = dup_samples | s
 
             self._build_mapping_file(samples, rename_dup_samples)
+            # TODO: addtl_processing_cmd needs to be passed to here and work
```

Comment:
All these comments seem a bit too much for me but I guess, please confirm, that make the code clearer for you, if that's the case, let's leave them. However, could you remove the here section, I guess it's implied when the comment is in that line. For example:
```
# TODO: addtl_processing_cmd needs to be passed to here and work 
# here.
```
can be: `# TODO: addtl_processing_cmd needs to be passed`
or  `# save final biom here` -> `# save final biom`

Reply:
Not a problem. I can remove them.